roles/endpoints/defaults/main.yml

```diff
@@ -238,6 +238,6 @@ endpoints:
       backend_api: 8043
       haproxy_api: 8042
     haproxy:
-      health_check: 'httpchk GET /healthcheck'
+      health_check: 'httpchk /'
```

Comment:
aodh can use the health check middleware provided by oslo. That can be a follow up PR though.